### PR TITLE
Order Detail

### PR DESCRIPTION
**Pull request by abbyfleming**
## Description
Order display now shows Name of product, description, quantity, line total, and order total. An "error" message will display if no payment types are added and a drop down list will display if there is a payment type.

## Related Ticket(s)
#9 #56 

## Problem to Solve
We needed to have the ability to see the quantity of items + the amount totaled.

## Changes
Did the success page change?  @whitneycormack @sales-ls21 

## Expected Behavior
Customer should be able to see the product, quantity, line total, and total. The customer should be able to select a payment and/or create a new payment.

## Steps to Test Solution
git fetch --all
git checkout afCartTemplate
rm -rf bang_app/migrations
rm -rf db.sqlite3
python manage.py makemigrations
python manage.py migrate
python manage.py runserver

* Register a new user
* Create **TWO** new products with **ONE** product having a quantity > 1
* Add **BOTH** products to the cart making sure **ONE** product has a quantity > 1
* Proceed to Checkout
* Create a New Payment Type
* Return to Checkout
* Success page?

## Documentation

- [ ] There is new documentation in this pull request that must be reviewed..
- [X] I added documentation for any new classes/methods

## Deploy Notes
Just to keep things consistent, start off with a fresh db. Details included in Steps to Test.

### CODE REVIEW

File: bangazon_website/bang_app/templates/order_detail_view.html [hunk 1]
```diff
@@ -1,57 +1,78 @@
...
-            <input type="text" class="form-control" name="billing_name" value="" placeholder="Enter Name on Card">
-        </div>
-
-        <button type="submit">Complete Checkout</button>
-    </form>
+	{# checks to see if the user has a currently active order. If they do, the order detail/checkout page is rendered with all appropriate affordances#}
+	{% if line_items %}
+
+	{# beginning of checkout form #}
+	<form action="{%url 'bang_app:close_order' %}" method="post">
```
Comment (abbyfleming) [inline bangazon_website/bang_app/templates/order_detail_view.html:14]: @sales-ls21 @whitneycormack Did this change?

**Comment by abbyfleming** [comment]
p.s. - Also removed some print statements. :)

**Comment by samphillips1879** [comment]
Looks great! 1 little comment though, after the payment type is created I think steve wanted it to redirect back to the order page. Currently it redirects to the /categories page

**Comment by samphillips1879** [comment]
Looks great 👍

**Comment by samphillips1879** [comment]
Looks even greater 👍

**Review by rtwhitfield84 — APPROVED** [review]
👍 💯

**Pull request merged**